src/sentry/monitors/validators.py

```diff
@@ -139,6 +139,9 @@ def validate(self, attrs):
             # crontab schedule must be valid
             if not croniter.is_valid(schedule):
                 raise ValidationError({"schedule": "Schedule was not parseable"})
+            # Do not support 6 or 7 field crontabs
+            if schedule.count(" ") > 4:
+                raise ValidationError({"schedule": "Only 5 field crontab syntax is supported"})
```

Comment:
is this input `.strip()`-ed?

Reply:
I think something somewhere is doing it, but let me add it just to be explicit 👍

Reply:
ah. it is.

Reply:
Did you find where it actually happens?

Reply:
no i just tested it on prod